flake8_future_import.py

```diff
@@ -10,29 +10,42 @@
 except ImportError as e:
     argparse = e
 
-from ast import NodeVisitor, PyCF_ONLY_AST, Str, Module
+import ast
 
 __version__ = '0.3.2'
 
 
-class FutureImportVisitor(NodeVisitor):
+class FutureImportVisitor(ast.NodeVisitor):
 
     def __init__(self):
         super(FutureImportVisitor, self).__init__()
         self.future_imports = []
+
         self._uses_code = False
+        self._uses_print = False
+        self._uses_division = False
+        self._uses_import = False
+        self._uses_str_literals = False
 
     def visit_ImportFrom(self, node):
         if node.module == '__future__':
             self.future_imports += [node]
-
-    def visit_Expr(self, node):
-        if not isinstance(node.value, Str) or node.value.col_offset != 0:
-            self._uses_code = True
+        else:
+            self._uses_import = True
 
     def generic_visit(self, node):
-        if not isinstance(node, Module):
+        if not isinstance(node, ast.Module):
             self._uses_code = True
+
+        if isinstance(node, ast.Str):
+            self._uses_str_literals = True
+        elif isinstance(node, ast.Print):
```

Comment:
I do these checks inside of `generic_visit` because for this file:

```python
print 'foo'
```

That should trigger both `uses_str_literals` and `uses_print`, but if I define `visit_Print` and `visit_Str`, it seems only one was ever getting called.